engine/src/WindowGLFW.cpp

```diff
@@ -49,4 +51,9 @@ void WindowGLFW::release() noexcept
     glfwDestroyWindow(m_window);
     glfwTerminate();
     Log::log("GLFW window release");
+}
+
+void Engine::Core::Renderering::WindowGLFW::getSize(unsigned short &width, unsigned short &height) const noexcept
+{
+    glfwGetWindowSize(m_window, reinterpret_cast<int *>(&width), reinterpret_cast<int *>(&height));
```

Comment:
An unsigned short is shorter than an int.
reinterpret_cast is a bad idea, since it would mean unallocated memory is being used.
A static_cast with copies and without pointers is probably a better idea.